databricks/koalas/generic.py

```diff
@@ -84,6 +84,19 @@ def abs(self):
         """
         return _spark_col_apply(self, F.abs)
 
+    def groupby(self, by):
+        from databricks.koalas.groupby import GroupBy
+        from databricks.koalas.series import Series
+        if isinstance(by, str):
+            by = [by]
+        elif isinstance(by, Series):
```

Comment:
check the anchor of the series is the same too.

Reply:
also, you should split that ino the SeriesGroupBy and the DataFrameGroupBy into `frames.py` and `series.py`?

Reply:
I'll split them when we actually see considerable divergence between them.